File: lib/models/salon.dart
class Salon {
  final String id;
  final String name;
  final String district;
  final String address;
  final double rating;
  final String openHours;
  final String imageAsset;

  const Salon({
    required this.id,
    required this.name,
    required this.district,
    required this.address,
    required this.rating,
    required this.openHours,
    this.imageAsset = '',
  });

  factory Salon.fromJson(Map<String, dynamic> json) {
    return Salon(
      id: json['id'] as String,
      name: json['name'] as String,
      district: json['district'] as String,
      address: json['address'] as String,
      rating: (json['rating'] as num).toDouble(),
      openHours: json['openHours'] as String,
    );
  }
}

File: backend/src/models/Salon.js
const { Schema, model } = require('mongoose');

const salonSchema = new Schema({
  name: { type: String, required: true },
  district: { type: String, required: true },
  address: { type: String, required: true },
  rating: { type: Number, default: 4.5 },
  openHours: { type: String, default: '08:00 - 21:00' },
});

module.exports = model('Salon', salonSchema);
